chainregistry.go

```diff
@@ -133,28 +132,25 @@ type chainControl struct {
 }
 
 // newChainControlFromConfig attempts to create a chainControl instance
-// according to the parameters in the passed lnd configuration. Currently three
+// according to the parameters in the passed configuration. Currently three
```

Comment:
a lot of the above fields can be replaced by holding a reference to the config

Reply:
The `chainregistry.Config`?

Reply:
I was thinking things like the signer, keyring etc was already in the config, but seems like that was wrong, so scratch that commetn.